package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "Blockly Developer Tools",
```

Comment:
Whoops.  Looks like this has to conform to [NPM conventions](https://docs.npmjs.com/getting-started/using-a-package.json). Use blockly-devtools